docs/source/conf.py

```diff
@@ -18,7 +18,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'AMD-RPP'
-copyright = '2020, Lokeswara Rao Bonta(MCW-AMD)'
+copyright = '2020 - 2022, Lokeswara Rao Bonta(MCW-AMD)'
```

Comment:
@kiritigowda This is different than the other copyrights. Is that okay? Seems like it's the only occurance though

Reply:
Should be ok

Reply:
Can be merged then